src/routes/docs/advanced/security/https/+page.markdoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: article
+title: HTTPS
+description: Learn how Appwrite Cloud enforces secure connections by enforcing HTTPS on all endpoints.
+---
+
+Appwrite Cloud serves all endpoints over an HTTPS connection by default. 
+Requests made through an unsecure HTTP connection will be redirected to.
+
+Redirected requests will show a `301` response status.
+
+```http
+HTTP/1.1 301 Moved Permanently
+Content-Type: application/json
+Location: https://cloud.appwrite.io/v1/<ENDPOINT>
+```
+
+Appwrite Cloud does not support HTTP, which is a common practice in modern development, because unencrypted
```

Comment:
I would also add a paragraph about our use of the HSTS header: https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security